### ISSUE TITLE
create fabric

### ISSUE THREAD

**Issue by BEY-joesfeir** [ionicthemes/ionic2-twitter-login#3]
how did you create a fabric_key and use it in your plugin

**Comment by djabif** [comment]
yes! the complete process is explained in this tutorial https://ionicthemes.com/tutorials/about/ionic-twitter-login
